src/app/beer_garden/garden.py

```diff
@@ -52,7 +52,9 @@ def get_gardens(include_local: bool = True) -> List[Garden]:
         All known gardens
 
     """
-    gardens = db.query(Garden)
+    # This is necessary for as long as local_garden is still needed. See the notes
+    # there for more detail.
+    gardens = db.query(Garden, filter_params={"connection_type__ne": "LOCAL"})
```

Comment:
I'm unsure why you're filtering for `"connection_type__ne"` here. Is this mongo magic?

Reply:
It's [mongoengine filter syntax](https://docs.mongoengine.org/guide/querying.html#query-operators).  The `db.query` call ultimate ends up doing:

```python
query_set = query_set.filter(**(kwargs.get("filter_params", {}))) 
```

Which in this case is essentially:
```python
queryset = Garden.objects.filter(connection_type__ne="LOCAL")
```

Reply:
Makes sense to me. My Google-Fu was not strong enough to find that syntax.